.github/workflows/upstream-js.yml

```diff
@@ -113,10 +113,13 @@ jobs:
         run: |
 
           migrateDependenciesJS() {
-              echo "Update JS dependencies to: ${PACKAGE_VERSION_JS}"
-              echo "Calling migration JS"
-              npx nx migrate @alfresco/js-api@${PACKAGE_VERSION_JS}
-              echo "Migration JS done"
+            echo "Update JS dependencies to: ${PACKAGE_VERSION_JS}"
+            echo "Calling migration JS"
+            npx nx migrate @alfresco/js-api@${PACKAGE_VERSION_JS}
+            cd lib/cli
```

Comment:
you can reduce the code if taking into brackets

```sh
(cd lib/cli && npx nx migrate @alfresco/js-api@${PACKAGE_VERSION_JS})
```

in that case you don't need "cd ../.."

Reply:
Changed